tests/WebAppCallsMicrosoftGraph/appsettings.json

```diff
@@ -1,14 +1,11 @@
 {
     "AzureAd": {
         "Instance": "https://login.microsoftonline.com/",
-        "Domain": "msidentitytesting.onmicrosofonline.com",
-        "TenantId": "7f58f645-c190-4ce5-9de4-e2b7acd2a6ab",
-        "ClientId": "56c9a633-236e-45ee-9af1-a53d9811fbd6",
-        // To call an API
-        "ClientSecret": "secret-goes-here",
+        "TenantId": "776c277d-e57f-4982-a3df-589bac9fadc6",
```

Comment:
can you make a new section for testing this so we don't lose our current app creds?

Reply:
done/ Thanks for the suggestion